ts/features/itwallet/presentation/details/components/ItwPresentationCredentialStatusAlert.tsx

```diff
@@ -98,30 +124,69 @@ const ItwPresentationCredentialStatusAlert = ({ credential }: Props) => {
     }
   };
 
-  if (status === "jwtExpiring") {
+  return (
+    <AlertToRender
+      status={status}
+      message={message}
+      credential={credential}
+      eidStatus={eidStatus}
+      isItwL3={isItwL3}
+      navigation={navigation}
+      offlineAccessReason={offlineAccessReason}
+      onTrack={trackCredentialAlertEvent}
+    />
+  );
```

Comment:
Can we simplify this? I don't see any advantage on splitting the logic in two components, we could return everything directly. What do you think?